cmd/xmidt-agent/main.go

```diff
@@ -266,6 +266,10 @@ func onStop(ws *websocket.Websocket, qos *qos.Handler, shutdowner fx.Shutdowner,
 		ws.Stop()
 		qos.Stop()
 		for _, c := range cancels {
```

Comment:
currently this triggers panics when a wrp handler returns a nil for `Cancel func() `group:"cancels"`

this bug wasn't caught because we weren't returning errors when fx's lifecycle Start/Stop encountered a panic
  - #124 fixes our fx lifecycle